src/index.js

```diff
@@ -148,6 +150,8 @@ export default class Headroom extends Component {
             this.eventListenerOptions
           )
       }
+    } else if (prevProps.pin !== this.props.pin) {
+      this.handleScroll()
```

Comment:
Is there any reason to put it in `else if` instead of `if`? Previous conditions check if props changed from `disabled: true` to `disabled: false` or the other way around. Those two cases are handled differently but it's unrelated to change of `pin` prop. So I think you could change it to a separate `if` instead of `else if`.